### PR TITLE
Issue 263: Speech recognition cues

### PR DESCRIPTION
**Pull request by JRustyHaner**
Implements speech recognition cues as described in #263

### CODE REVIEW

**Comment by JRustyHaner** [comment]
- Change icon to white when listening
- Change language when doing a button based question
- make sure each session type has a prompt

**Review by andrewtackett — CHANGES_REQUESTED** [review]
need to change color of mic on not transcribing, have diff text for button trial, make sure appears in both learningsession and assessmentsession

**Comment by JRustyHaner** [comment]
@andrewtackett changed according to yesterday's discussion about the icon, message phrasing, and going to an on state for microphone icon during silence detected.

**Comment by JRustyHaner** [comment]
@andrewtackett  fixed spacing and the issue discussed where icon was being displayed when voice prompts weren't required.

**Comment by JRustyHaner** [comment]
@andrewtackett I compared the changes to the original and it should match up

File: mofacts/client/views/experiment/card.js [hunk 1]
```diff
@@ -1163,13 +1190,7 @@ async function writeCurrentToScrollList(userAnswer, isTimeout, simCorrect, justA
     // Timeout
     userAnswerWithTimeout = '';
     isCorrect = false;
-    historyUserAnswer = 'You didn\'t answer in time.';
-  } else if (typeof simCorrect === 'boolean') {
```
Comment (andrewtackett) [inline mofacts/client/views/experiment/card.js]: These lines are still deleted

File: mofacts/client/views/experiment/card.js [hunk 2]
```diff
@@ -2104,8 +2126,11 @@ async function processLINEAR16(data) {
         phraseHints.push(curChar);
       }
     } else {
+
+      //look at this for transcription prompt!
```
Comment (andrewtackett) [inline mofacts/client/views/experiment/card.js]: Remove this comment

File: mofacts/client/views/experiment/card.js [hunk 3]
```diff
@@ -2195,6 +2221,8 @@ function makeGoogleSpeechAPICall(request, speechAPIKey, answerGrammar) {
         }
       }
     } else {
+      //Change graphic and message
```
Comment (andrewtackett) [inline mofacts/client/views/experiment/card.js]: Remove this comment

File: mofacts/client/views/experiment/card.js [hunk 4]
```diff
@@ -2351,7 +2379,7 @@ function startUserMedia(stream) {
         if (resetMainCardTimeout && timeoutFunc) {
           if (Session.get('recording')) {
             console.log('voice_start resetMainCardTimeout');
-            resetMainCardTimeout();
+
```
Comment (andrewtackett) [inline mofacts/client/views/experiment/card.js]: Don't delete this

File: mofacts/client/views/experiment/card.js [hunk 5]
```diff
@@ -2375,7 +2403,8 @@ function startRecording() {
   if (recorder) {
     Session.set('recording', true);
     recorder.record();
-    console.log('RECORDING START');
+    console.log('RECORDING START');    
+    resetMainCardTimeout();
```
Comment (andrewtackett) [inline mofacts/client/views/experiment/card.js]: Don't add this here

**Review by andrewtackett — CHANGES_REQUESTED** [review]

**Comment by JRustyHaner** [comment]
@andrewtackett  took me a minute, but its showing no subtractions on my end now.

**Review by andrewtackett — APPROVED** [review]
Looks good

**Pull request merged**